exercises/perfect-numbers/README.md

```diff
@@ -0,0 +1,16 @@
+Determine if a number is perfect, abundant, or deficient based on
```

Comment:
The readme needs to be generated from the templates with the configlet. You can find more information in the `CONTRIBUTING.md` and in [Generating a README](https://github.com/exercism/docs/blob/master/language-tracks/exercises/anatomy/readmes.md#generating-a-readme)